### PR TITLE
Multiple minor changes to layout (Fixes #125)

### LINKED ISSUE #125: Multiple minor page layout and information changes
As per George's request, following a discussion:

**Nightly:**
Remove release details section
Make checksum and release date their own TD elements

**Archive:**
Release details block remove

**Footer:**
Got an issue with the website? link to issues on repo

### PR DESCRIPTION
**Pull request by Joe-Brady**
On [Staging](https://staging.adoptopenjdk.net)
As per George's request, following a discussion:

**Nightly:**
Removed release details section
Made checksum and release date their own columns for correct alignment

**Archive:**
Release details block removed

**Footer:**
Links for people who spot issues in website

### CODE REVIEW

File: src/handlebars/partials/footer.handlebars [hunk 1]
```diff
@@ -25,16 +25,8 @@
...
-  <div style="margin-top: 1.5rem;" id="google_translate_element"></div>
-  <script type="text/javascript">
-    function googleTranslateElementInit() {
-      new google.translate.TranslateElement({pageLanguage: 'en', includedLanguages: 'en,pt,zh-CN', layout: google.translate.TranslateElement.InlineLayout.SIMPLE}, 'google_translate_element');
-    }
-  </script>
-  <script type="text/javascript" src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
-  -->
+  <br>
+  <p style="line-height: 1rem;"><strong>Spotted an issue with the website?</strong><br><a target="_blank" class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/issues">Raise a GitHub issue</a> or <a target="_blank" class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md">contribute a change!</a></p>
```
Comment (gdams) [inline src/handlebars/partials/footer.handlebars]: can the contribute a change bit be put in the right hand corner, I don't like the way the footer is getting thicker

Comment (Joe-Brady) [inline src/handlebars/partials/footer.handlebars]: Not sure, looks a bit weird?

screen shot 2017-06-02 at 10 25 37

Comment (Joe-Brady) [inline src/handlebars/partials/footer.handlebars]: But I know what you mean about the footer getting thicker....

Comment (Joe-Brady) [inline src/handlebars/partials/footer.handlebars]: @karianna What's your thoughts on the footer on [staging](https://staging.adoptopenjdk.net)? I'm not sure if it's an issue, or whether slightly longer footers are pretty standard?

I'm trying to think of an alternative place to put the link on every page (already adding it to the Get Involved page, but that's not enough)... no ideas just yet!

Comment (karianna) [inline src/handlebars/partials/footer.handlebars]: I'm OK with a long footer, it's pretty common these days.

**Pull request merged**